packages/plan-form-core/src/formHelpers/activitiesLookup.ts

```diff
@@ -0,0 +1,1251 @@
+import {
+  BCC_CODE,
+  INVESTIGATION,
+  MEDIUM_PRIORITY,
+  IRS_CODE,
+  ROUTINE,
+  BEDNET_DISTRIBUTION_CODE,
+  BLOOD_SCREENING_CODE,
+  CASE_CONFIRMATION_CODE,
+  APPLICABILITY_CONDITION_KIND,
+  PLAN_ACTIVATION_TRIGGER_NAME,
+  NAMED_EVENT_TRIGGER_TYPE,
+  CREATE_TYPE,
+  MDA_ADHERENCE_CODE,
+  MDA_POINT_DISPENSE_CODE,
+  RACD_REGISTER_FAMILY_CODE,
+  LARVAL_DIPPING_CODE,
+  MOSQUITO_COLLECTION_CODE,
+  MDA_POINT_ADVERSE_EFFECTS_CODE,
+  PRODUCT_CHECK_ACTIVITY_CODE,
+  PRODUCT_CHECK_CODE,
+  FIX_PRODUCT_PROBLEM_ACTIVITY_CODE,
+  FIX_PRODUCT_PROBLEMS_CODE,
+  RECORD_GPS_ACTIVITY_CODE,
+  RECORD_GPS_CODE,
+  SERVICE_POINT_CHECK_ACTIVITY_CODE,
+  SERVICE_POINT_CHECK_CODE,
+} from './constants/stringConstants';
+import {
+  BCC_ACTIVITY_DESCRIPTION,
+  BCC_ACTIVITY,
+  BCC_GOAL_DESCRIPTION,
+  BCC_GOAL_MEASURE,
+  IRS_ACTIVITY_DESCRIPTION,
+  IRS_ACTIVITY,
+  IRS_GOAL_DESCRIPTION,
+  IRS_GOAL_MEASURE,
+  BEDNET_ACTIVITY_DESCRIPTION,
+  BEDNET_ACTIVITY,
+  BEDNET_GOAL_MEASURE,
+  BLOOD_SCREENING_ACTIVITY_DESCRIPTION,
+  BLOOD_SCREENING_ACTIVITY,
+  BLOOD_SCREENING_GOAL_MEASURE,
+  CASE_CONFIRMATION_ACTIVITY_DESCRIPTION,
+  CASE_CONFIRMATION_ACTIVITY,
+  CASE_CONFIRMATION_ACTIVITY_GOAL_MEASURE,
+  REGISTER_FAMILY_ACTIVITY_DESCRIPTION,
+  REGISTER_FAMILY_ACTIVITY,
+  REGISTER_FAMILY_ACTIVITY_GOAL_MEASURE,
+  LARVAL_DIPPING_ACTIVITY_DESCRIPTION,
+  LARVAL_DIPPING_ACTIVITY,
+  LARVAL_DIPPING_GOAL_MEASURE,
+  MOSQUITO_COLLECTION_ACTIVITY_DESCRIPTION,
+  MOSQUITO_COLLECTION_ACTIVITY,
+  MOSQUITO_COLLECTION_GOAL_MEASURE,
+  MDA_POINT_ADVERSE_EFFECT_ACTIVITY_DESCRIPTION,
+  MDA_POINT_ADVERSE_EFFECT_COLLECTION_GOAL,
+  MDA_POINT_DISPENSE_ACTIVITY_DESCRIPTION,
+  MDA_POINT_DISPENSE_COLLECTION_GOAL,
+  PRODUCT_CHECK_ACTIVITY_DESCRIPTION,
+  PRODUCT_CHECK_GOAL_MEASURE,
+  FIX_PRODUCT_PROBLEM_ACTIVITY_DESCRIPTION,
+  FIX_PRODUCT_PROBLEM_GOAL_MEASURE,
+  RECORD_GPS_ACTIVITY_DESCRIPTION,
+  RECORD_GPS_GOAL_MEASURE,
+  SERVICE_POINT_CHECK_ACTIVITY_DESCRIPTION,
+  SERVICE_POINT_CHECK_GOAL_MEASURE,
+} from './constants/lang';
+import { PlanActivities } from './types';
+import { GoalUnit } from './constants/enumsAndCodeConstants';
+
+/** default plan activities */
+export const planActivities: PlanActivities = {
```

Comment:
I'd like us to find a way to make this into a configuration in the future.  So that adding/changing the templates is more straight forward